chats/apps/api/v1/internal/projects/viewsets.py

```diff
@@ -75,3 +78,18 @@ def put(
                 status.HTTP_400_BAD_REQUEST,
             )
         return Response({"Detail": "Updated"}, status.HTTP_200_OK)
+
+    @action(detail=True, methods=["GET"], permission_classes=[IsAuthenticated, SectorAnyPermission])
+    def online_status(self, *args, **kwargs):
+        instance = self.get_object()
+
+        if instance.status == "ONLINE":
+            instance.status = ProjectPermission.STATUS_OFFLINE
+            instance.save()
+        else:
+            instance.status = ProjectPermission.STATUS_ONLINE
+            instance.save()
+            
+        serializer = serializers.OnlineStatusReadSerializer(instance=instance)
+
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
As this is a super simple API, I believe it doesn't need a serializer. just return:
```py
return Response(dict(status=instance.status), status=status.HTTP_200_OK)
```